### PR TITLE
[KOB-6220] Add Applitools into kobiton/samples

### PR DESCRIPTION
**Pull request by oanhpham173**
Add Applitools into kobiton/samples

### CODE REVIEW

File: applitools/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+mkdir kobiton-samples
+cd kobiton-samples
+git clone https://github.com/kobiton/samples.git
+cd samples/applitools
+npm install
+```
+
+### Getting started with Kobiton
+
+- Go to [Kobiton Devices page](https://portal-test.kobiton.com/devices)
```
Comment (MinhLe97) [inline applitools/README.md:19]: https://portal.kobiton.com/devices

File: applitools/test/android-web-test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        input.waitForDisplayed()
+        await eyes.check('Google Home', Target.window())
+
+        await input.setValue('Kobiton')
+        await eyes.check('Drop down list', Target.window())
+
+        await eyes.close()
+    })
+  })
+
```
Comment (khanhdodang) [inline applitools/test/android-web-test.js:36]: Add a newline

File: applitools/wdio.conf.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    baseUrl: 'https://portal.kobiton.com',
+    hostname: 'api.kobiton.com',
+    waitforTimeout: 10000,
+    connectionRetryTimeout: 90000,
+    connectionRetryCount: 3,
+    reporters: ['spec'],
+    port: 443,
+    maxInstances: 1,
+}
+
```
Comment (khanhdodang) [inline applitools/wdio.conf.js:36]: Add a newline

File: applitools/test/android-web-test.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+} = require('@applitools/eyes-webdriverio');
+  
+describe('WebdriverIO', () => {
+    it('should run scripts in kobiton and send results of mobile simulators to applitools server', async () => {
+        const runner = new VisualGridRunner(1)
+        const eyes = new Eyes(runner);
+
+        const configuration = new Configuration();
+        configuration.setApiKey('APPLITOOLS_API_KEY') // You can get your api key from the Applitools dashboard
+        //Add mobile emulation devices with Portrait mode
```
Comment (khanhdodang) [inline applitools/test/android-web-test.js:17]: We use real devices, pls remove this line

File: applitools/test/android-web-test.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+const {
```
Comment (khanhdodang) [inline applitools/test/android-web-test.js:1]: Check the format

File: applitools/test/android-web-test.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    VisualGridRunner,
+    Eyes,
+    Configuration,
+    DeviceName,
+    ScreenOrientation,
+    Target
+} = require('@applitools/eyes-webdriverio');
+  
+describe('WebdriverIO', () => {
+    it('should run scripts in kobiton and send results of mobile simulators to applitools server', async () => {
```
Comment (khanhdodang) [inline applitools/test/android-web-test.js:11]: remove `of mobile simulators`

**Comment by khanhdodang** [comment]
Remove applitools/package-lock.json

File: applitools/README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+- Update the global variables `KOBITON_USERNAME` and `KOBITON_API_KEY` using bash shell (you can set in the user directories .bash_profile) by adding the following to new lines of this file or replace `user` & `key` value in the config file (./config/wdio.conf.js):
+
+```
+export KOBITON_USERNAME=''
+export KOBITON_API_KEY=''
+```
+
+- Update `capabilities` at the config file (.wdio.conf.js) follow up above information to indicate your expected testing device
+
+- Replace `APPLITOOLS_API_KEY` by the key from your Applitools dashboard at the test file (android-web-test)
```
Comment (khanhdodang) [inline applitools/README.md:34]: Add URL of Applitools

File: applitools/test/android-web-test.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        const runner = new VisualGridRunner(1)
+        const eyes = new Eyes(runner);
```
Comment (khanhdodang) [inline applitools/test/android-web-test.js:13]: c

File: applitools/test/android-web-test.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+        const configuration = new Configuration();
```
Comment (khanhdodang) [inline applitools/test/android-web-test.js:15]: Remove `;`

File: applitools/README.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
+# Javascript samples using applitools
```
Comment (HienNguyenT) [inline applitools/README.md:1]: It should be "Applitools".

File: applitools/README.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+## I. Setup environment on Mac
+
+### Install the applitool samples
```
Comment (HienNguyenT) [inline applitools/README.md:5]: It should be "Applitools".

File: applitools/wdio.conf.js [hunk 12]
```diff
@@ -0,0 +1,36 @@
+exports.config = {
+    user: process.env.KOBITON_USERNAME,
+    key: process.env.KOBITON_API_KEY,
+    runner: 'local',
```
Comment (khanhdodang) [inline applitools/wdio.conf.js:4]: Why do we define runner = local?

File: applitools/README.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+Once you have everything set up, you can run the example test simply by running one of the following command:
+
+```
+npm run test
+```
+## III. Report on the test results
+
+When you see your test 'Complete' on the Terminal, 
+ - You can access https://portal.kobiton.com/sessions to get your test results.
+ - You can access to https://eyes.applitools.com/app/results to get your test results
```
Comment (HienNguyenT) [inline applitools/README.md:47]: Please check the grammar here.

File: applitools/package.json [hunk 14]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "applitools",
+  "version": "1.0.0",
+  "description": "This is a sample for applitools",
```
Comment (HienNguyenT) [inline applitools/package.json:4]: It should be "Applitools".

File: applitools/test/android-web-test.js (hunk 6 above)
Comment (HienNguyenT) [reply applitools/test/android-web-test.js:11]: kobiton -> Kobiton

File: applitools/test/android-web-test.js [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+        configuration.setApiKey('APPLITOOLS_API_KEY') // You can get your api key from the Applitools dashboard
+        //Add mobile emulation devices with Portrait mode
+        configuration.addDeviceEmulation(DeviceName.iPhone_4, ScreenOrientation.PORTRAIT);
+        configuration.addDeviceEmulation(DeviceName.iPhone_X, ScreenOrientation.PORTRAIT);
+        configuration.addDeviceEmulation(DeviceName.Pixel_2, ScreenOrientation.PORTRAIT);
```
Comment (HienNguyenT) [inline applitools/test/android-web-test.js:20]: Why do we need to add these lines?

File: applitools/package.json (hunk 14 above)
Comment (oanhpham173) [reply applitools/package.json:4]: this is a name of test, company name => keep this

**Pull request merged**